tests/distributed/test_rolling_update_container_runtime/test_rolling_update.py

```diff
@@ -0,0 +1,111 @@
+import os
+import time
+
+import numpy as np
+import pytest
+
+from jina import Document, Client, __default_host__
+from jina.logging.logger import JinaLogger
+from daemon.clients import JinaDClient
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+compose_yml = os.path.join(cur_dir, 'docker-compose.yml')
+
+HOST = __default_host__
+JINAD_PORT = 8000
+REST_PORT_DBMS = 9000
+REST_PORT_QUERY = 9001
+DUMP_PATH = '/jinad_workspace/dump'
+
+logger = JinaLogger('test-dump')
+client = JinaDClient(host=HOST, port=JINAD_PORT)
+
+SHARDS = 3
+EMB_SIZE = 10
+
+
+@pytest.fixture
+def executor_images():
+    dbms_dir = os.path.join(cur_dir, 'pods', 'dbms')
+    dbms_docker_file = os.path.join(dbms_dir, 'Dockerfile')
+    os.system(f"docker build -f {dbms_docker_file} -t dbms-executor {dbms_dir}")
+    query_dir = os.path.join(cur_dir, 'pods', 'query')
+    query_docker_file = os.path.join(query_dir, 'Dockerfile')
+    os.system(f"docker build -f {query_docker_file} -t query-executor {query_dir}")
+    time.sleep(3)
+    yield
+    os.system(f"docker rmi $(docker images | grep 'dbms-executor')")
+    os.system(f"docker rmi $(docker images | grep 'query-executor')")
+
+
+def _create_flows():
+    workspace_id = client.workspaces.create(paths=[cur_dir])
+    dbms_flow_id = client.flows.create(
+        workspace_id=workspace_id,
+        filename='flow_dbms.yml',
+        envs={'JINAD_WORKSPACE': f'/tmp/jinad/{workspace_id}'},
+    )
+    query_flow_id = client.flows.create(
+        workspace_id=workspace_id,
+        filename='flow_query.yml',
+        envs={'JINAD_WORKSPACE': f'/tmp/jinad/{workspace_id}'},
+    )
+    return dbms_flow_id, query_flow_id, workspace_id
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_dump_dbms_remote(executor_images, docker_compose):
```

Comment:
Can u parametrize this to show what works and what doesn't? 

`replicas = 1,2`
`parallel = 1,2`